src/spec.c

```diff
@@ -1195,6 +1195,11 @@ static inline uint16_t TranslateMask(uint64_t maskPart, t_fieldIndex *translatio
 uint16_t IndexSpec_TranslateMaskToFieldIndices(const IndexSpec *sp, t_fieldMask mask, t_fieldIndex *out) {
   uint16_t count = 0;
   const uint8_t LOW_OFFSET = 0;
+  if (mask == RS_FIELDMASK_ALL) {
```

Comment:
Is it possible to add a test to cover this case?

Reply:
We have a test that causes mask to be `RS_FIELDMASK_ALL`.
Problem is the side affects of the test didn't cause a crash so we didn't have a way to detect them.

Looking again at the code, maybe a better solution would be to check the size of the translation table and not exceed it regardless of the mask we get.